package.json

```diff
@@ -20,6 +20,18 @@
       "import": "./lib/browser/index.mjs",
       "default": "./lib/browser/index.js"
     },
+    "./http": {
+      "types": "./lib/core/http.d.ts",
+      "require": "./lib/core/http.js",
+      "import": "./lib/core/http.mjs",
+      "default": "./lib/core/http.mjs"
+    },
+    "./graphql": {
+      "types": "./lib/core/graphql.d.ts",
+      "require": "./lib/core/graphql.js",
+      "import": "./lib/core/graphql.mjs",
+      "default": "./lib/core/graphql.mjs"
```

Comment:
We seem to have inconsistent `default` condition across exports:

- `/http`, `/graphql` and `/node` list the ESM build as the default;
- `.`, `/browser` and `/native` list the CJS build as the default. 

Is this something we should unify as a part of this change as well?

Reply:
we can change the `default` for `./http` and `./graphql`, but changing the defaults for other keys may be a breaking change

Reply:
I was mainly suggesting the former. As we are in this weird CJS/ESM situation, the default must remain CJS. Would you please make the change?

Reply:
done